old-ui/css.js

```diff
@@ -18,6 +18,7 @@ var cssFiles = {
   'react-css': fs.readFileSync(path.join(__dirname, '..', 'node_modules', 'react-select', 'dist', 'react-select.css'), 'utf8'),
   'dropdowns.css': fs.readFileSync(path.join(__dirname, '/app/css/dropdowns.css'), 'utf8'),
   'app-bar.css': fs.readFileSync(path.join(__dirname, '/app/css/app-bar.css'), 'utf8'),
+  'rif.css': fs.readFileSync(path.join(__dirname, '/app/css/rif.css'), 'utf8'),
```

Comment:
This we will need to delete it, so each page has its own css, in PR #3 i can integrate this change.

Reply:
I think you didn't understand the logic, that file is generated by gulp with a task, all the different css files inside the folder rif end merged in that rif.css file, then this css.js reads that merged file and it uses it as it does with the other files too.

Reply:
BTW, this file is so critical that making changes on the css.js is not an option.

Reply:
Oh! Nice, ok then!